src/App.tsx

```diff
@@ -1,54 +1,83 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
 import './App.scss';
+import { PersonList } from './components/PersonList/PersonList';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const [isPersonListActive, setIsPersonListActive] = useState(false);
```

Comment:
try to implement this part:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/16cf7391-f702-46ea-9f1b-4e17e542c90d)

ask in the chat if you need help